crap-advisor-frontend/src/core/components/Map/index.css

```diff
@@ -1,5 +1,4 @@
 .map {
-    position: fixed;
-    top: 0;
-    left: 0;
+    width: 100%;
+    height: 100%;
```

Comment:
Perhaps you should add
`max-width: 100%; 
max-height: 100%; 
min-width: 100%; 
min-height: 100% `
 to the correct display on some devices

Reply:
Sadly min- guys made that the map has become larger than my screen

Reply:
Uh, okay, max- guys is good enough. I'll mark this conversation as resolved